src/app/_components/home/CotamPeople.tsx

```diff
@@ -22,36 +26,35 @@ const CotamPeople = () => {
       icon: <Hand className={'size-4'} />,
       label: 'IT 업계분들과 교류하고 싶어요',
       backgroundColor: 'bg-cotam-blue-80',
-      delay: 500,
+      delay: 450,
     },
     {
       icon: <ArrowUp className={'size-4'} />,
       label: '성장하고 싶어요',
       backgroundColor: 'bg-cotam-blue-70',
-      delay: 700,
+      delay: 600,
     },
     {
       icon: <Scroll className={'size-4'} />,
       label: '생산적인 주말을 보내고 싶어요',
       backgroundColor: 'bg-cotam-blue-50',
-      delay: 900,
+      delay: 750,
     },
     {
       icon: <Target className={'size-4'} />,
       label: '목표를 달성하고 싶어요',
       backgroundColor: 'bg-cotam-blue-40',
-      delay: 1100,
+      delay: 900,
     },
   ];
 
   return (
     <section
       className={`sticky top-0 flex flex-col gap-9 bg-cotam-blue-100 px-10 py-[60px] ${ZIndex.home.thirdLayer}`}>
       <p className="break-keep text-white galmuri11-headline-1">이런 사람들과 함께하고 있어요.</p>
-      <ul className="flex flex-col flex-wrap gap-3 sm:flex-row">
-        {TopMoveBoxList.map((topMoveBox) => (
-          <TopMoveBox key={topMoveBox.label} {...topMoveBox} />
-        ))}
+      <ul className="flex h-full min-h-[268px] flex-col flex-wrap gap-3 sm:flex-row">
```

Comment:
```suggestion
      <ul className="flex h-full flex-col flex-wrap gap-3 sm:flex-row">
```
코멘트로 언급드렸던거, 여기 min-h-[268px] 추가로 발생하네요

Reply:
확인 감사합니다! 이거 수정해서 올릴게요!